lib/ex_css_modules/ex_css_modules.ex

```diff
@@ -40,7 +40,7 @@ defmodule ExCSSModules do
       true ->
         (filename <> ".json")
         |> File.read!()
-        |> Poison.decode!()
+        |> json_library().decode!()
```

Comment:
Is it safe to assume alternative JSON libs all have the `decode!/1` fun?

Reply:
Tested with [Jason](https://github.com/michalmuskala/jason) and work as expected.